### DISCUSSION TITLE
1.4 Release

### DISCUSSION THREAD

**Discussion by muit** [PipeRift/SaveExtension#66] [General]
Hi!

[Release 1.4](https://github.com/PipeRift/SaveExtension/releases/tag/1.4) is out and with it, everybody is welcomed to post here questions about the release or issues found.

Thanks for using the plugin :)

**Comment by Dan7210** [comment]
Hey so does this work with UE 4.26?

**Reply by vallamost** [reply]
Not at this time, looks like source needs to be built to support 4.26 :(

**Comment by dagouxiong20192019** [comment]
I have no the Plugins folder!!!
